esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -425,7 +425,7 @@ void myDisplayRecognitionError (pANTLR3_BASE_RECOGNIZER recognizer,pANTLR3_UINT8
                 // correct stream. Then we can see that the token we are looking at
                 // is just something that should not be there and throw this exception.
                 //
-                if  (tokenNames == NULL)
+                if  (tokenNames == nullptr)
```

Comment:
A small question on style.  Elsewhere we don't use brackets {} if the conditional only has one action afterwards.  Should we remove the brackets here and just use indentation?

Reply:
ummm good eye... in this case, I'm not sure it's worth changing. @Michael-Gardner